src/vs/workbench/contrib/chat/browser/chatElicitationRequestPart.ts

```diff
@@ -23,9 +24,11 @@ export class ChatElicitationRequestPart extends Disposable implements IChatElici
 		public readonly subtitle: string | IMarkdownString,
 		public readonly acceptButtonLabel: string,
 		public readonly rejectButtonLabel: string,
-		public readonly accept: () => Promise<void>,
+		// True when the primary action is accepted, otherwise the action that was selected
+		public readonly accept: (value: IAction | true) => Promise<void>,
```

Comment:
`IChatElicitationRequest.accept` uses a `boolean`?

Reply:
https://github.com/microsoft/vscode/blob/25b0bbfc6e5f906e1d7dc5541bfff2dc527e10a7/src/vs/workbench/contrib/chat/browser/chatContentParts/chatElicitationContentPart.ts#L58-L77

When I click `Allow`, IE accept primary action, `data` is `true`.

<img width="1175" height="509" alt="Screenshot 2025-08-27 at 11 33 17 AM" src="https://github.com/user-attachments/assets/0f6ba3e9-aa5d-4b4a-aff4-b97a5f108fac" />

When I click one of `moreActions`, `data` is an `IAction`

<img width="975" height="379" alt="Screenshot 2025-08-27 at 11 33 02 AM" src="https://github.com/user-attachments/assets/3eae0fe0-5616-4434-8143-323479b71d9d" />